tests/ga4gh/broker/test_server.py

```diff
@@ -0,0 +1,69 @@
+import mongomock
+
+from flask import Flask
+
+from foca.models.config import Config
+from foca.models.config import MongoConfig
+
+from broker.ga4gh.broker.server import (
+    postRepositories
+)
+
+INDEX_CONFIG = {
+    'keys': [('id', -1)]
+}
+INDEX_CONFIG_USERS = {
+    'keys': [('uid', -1)]
+}
+
+COLLECTION_CONFIG = {
+    'indexes': [INDEX_CONFIG],
+}
+
+COLLECTION_CONFIG_USERS = {
+    'indexes': [INDEX_CONFIG_USERS],
+}
+
+DB_CONFIG = {
+    'collections': {
+        'repositories': COLLECTION_CONFIG,
+        'builds': COLLECTION_CONFIG,
+        'subscriptions': COLLECTION_CONFIG,
+        'users': COLLECTION_CONFIG_USERS
+    },
+}
+
+MONGO_CONFIG = {
+    'host': 'mongodb',
+    'port': 27017,
+    'dbs': {
+        'brokerStore': DB_CONFIG,
+    },
+}
+ENDPOINT_CONFIG = {
```

Comment:
Move all the constant mock objects to a separate file `mock.py` and import them individually.